server_db/tests/userTest.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-undef */
+/* eslint-disable prefer-destructuring */
+import { expect } from 'chai';
+import { describe } from 'mocha';
+import supertest from 'supertest';
+import server from '../../server';
+
+const api = supertest(server);
+
+describe('tests for user controller', async () => {
+  describe('/POST create user', () => {
+    it('should create a new user', (done) => {
+      const user = {
+        firstName: 'Julius',
+        lastName: 'Ngwu',
+        email: 'julius@gmail.com',
+        password: '123def',
+        phoneNumber: '09088776654',
+        type: 'staff',
+        isAdmin: true,
+      };
+      api.post('/api/v1/auth/signup')
+        .send(user)
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          expect(res.body).to.have.property('status');
+          expect(res.body.status).to.equal(201);
+          expect(res.body).to.have.property('data');
+          expect(res.body.data).to.have.property('token');
+          expect(res.body.data).to.have.property('id');
+          expect(res.header).to.have.property('x-access-token');
+          done();
+        });
+    });
+
+    it('should not create a new user', (done) => {
```

Comment:
It should not create a new user? why, you need to give reasons

Reply:
Alright
I'm making the changes right away